modules/xmpp/moderator.js

```diff
@@ -210,11 +210,12 @@ Moderator.prototype.createConferenceIq = function() {
                 value: this.options.conference.minBitrate
             }).up();
     }
-    if (this.options.conference.openSctp !== undefined) {
+    if (this.options.conference.openBridgeChannel !== undefined) {
         elem.c(
             'property', {
+                // TODO: Should be renamed to openBridgeChannel
```

Comment:
Not really, the JVB option is ok as openSctp (though I'm not a fan of the name) because it just tells the bridge to open a datachannel, and that still stands.